proceeding/text/introduction.tex

```diff
@@ -9,4 +9,18 @@ \section{Introduction}
 \item Because of an increasing interest in the field, there is a need for a hub of VHE data (GeV, TeV) across many different catalogs.
 This is what gamma-sky.net was created for.
 
+\pagebreak
```

Comment:
Remove pagebreak.
It's very uncommon to manually put the pagebreaks.
If at all, this should be done at the very end, after the paper is final.
For now, just let it flow however LaTeX layouts the paper.